src/main/java/org/dependencytrack/persistence/TagQueryManager.java

```diff
@@ -276,6 +288,13 @@ public void deleteTags(final Collection<String> tagNames) {
                             is missing the %s or %s permission.""".formatted(row.notificationRuleCount(),
                             Permissions.SYSTEM_CONFIGURATION, Permissions.SYSTEM_CONFIGURATION_UPDATE));
                 }
+
+                if (row.vulnerabilityCount() > 0 && !hasPortfolioManagementUpdatePermission) {
+                    errorByTagName.put(row.name(), """
+                            The tag is assigned to %d vulnerabilities, but the authenticated principal \
+                            is missing the %s or %s permission.""".formatted(row.vulnerabilityCount(),
+                            Permissions.PORTFOLIO_MANAGEMENT, Permissions.PORTFOLIO_MANAGEMENT_UPDATE));
+                }
```

Comment:
This should be `VULNERABILITY_MANAGEMENT` / `VULNERABILITY_MANAGEMENT_UPDATE` as per: https://github.com/DependencyTrack/hyades-apiserver/blob/d1c2b70e3a96f03da9c48246908d59ce7b29cecb/src/main/java/org/dependencytrack/resources/v1/VulnerabilityResource.java#L743-L744